contrib/building.md

```diff
@@ -0,0 +1,280 @@
+# Working With Realm-JS
+
+
+<!--ts-->
```

Comment:
Does this comment have a particular meaning / function?
Also, is the following list up-to-date?